xarray/tests/test_accessor_dt.py

```diff
@@ -423,22 +423,17 @@ def test_calendar_cftime(data) -> None:
 
 
 @requires_cftime
-def test_calendar_cftime_2D(data) -> None:
```

Comment:
The `data` fixture is not needed, since it is overwritten immediately, and otherwise causes this test to be run seven times (one for each cftime calendar, despite this testing on datetime64 data).